### ISSUE TITLE
Cordova 7.1 plugin install error

### ISSUE THREAD

**Issue by Amiga4ever** [sunnycupertino/cordova-plugin-admob-simple#42]
here's the log:
> cordova plugin add https://github.com/sunnycupertino/cordova-plugin-admob-simple
Failed to fetch plugin https://github.com/sunnycupertino/cordova-plugin-admob-simple via registry.
Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
Error: cmd: Command failed with exit code 1 Error output:
npm ERR! code ENOGIT
npm ERR! No git binary found in $PATH
npm ERR!
npm ERR! Failed using git.
npm ERR! Please check if you have git installed and in your PATH.

Basically I can't install plugin on fresh CORDOVA installation. Any ideas?

**Comment by sunnycupertino** [comment]
Thank you for pointing this out. Please use
cordova plugin add cordova-plugin-admob-simple
